src/Conversation.php

```diff
@@ -172,7 +173,7 @@ public function stop()
      */
     public function cancel()
     {
-        return $this->updateStatus('cancelled');
+        return ($this->updateStatus('cancelled') && $this->clear());
```

Comment:
If a stop the conversation I can leave conversation data inside instead to reset everything.

https://github.com/akalongman/php-telegram-bot/pull/104#issuecomment-193411287
I my last comment i said that could be a nice idea to have a var for each column of the database instead of store everything inside the '$this->conversation' var
something like :
```
protected id;
protected status;
ecc

```
This can be useful because after updating we can just update the status var inside the class.
Do you think that this can work?